### ISSUE TITLE
query returned wrong number of labels: 1, 0

### ISSUE THREAD

**Issue by wenerme** [timescale/timescale-prometheus#189]
* exporter
  * https://github.com/wrouesnel/postgres_exporter
  * node_exporter
*  one timescale-prometheus - docker installed - latest
* tow prometheus
  * `prom-a` one for collect
  * `prom-b` one for read by grafana

but query `process_start_time_seconds` will failed on `prom-b`, `prom-a` can execute correctly

```
remote_read: remote server http://10.1.1.3:9202/read returned HTTP status 500 Internal Server Error: query returned wrong number of labels: 1, 0
```

https://github.com/timescale/timescale-prometheus/blob/6693256251629559a609f9e7f56131d6107eba2f/pkg/pgmodel/sql_reader.go#L252

both node_exporter and postgres_exporter has process_start_time_seconds metric

error log

```json
{
  "caller": "read.go:45",
  "err": "query returned wrong number of labels: 1, 0",
  "level": "warn",
  "msg": "Error executing query",
  "query": {
    "queries": [
      {
        "start_timestamp_ms": 1596877133800,
        "end_timestamp_ms": 1596881033800,
        "matchers": [
          { "name": "__name__", "value": "process_start_time_seconds" }
        ],
        "hints": {
          "step_ms": 14000,
          "start_ms": 1596877133800,
          "end_ms": 1596881033800
        }
      }
    ]
  },
  "storage": "PostgreSQL",
  "ts": "2020-08-08T10:03:53.854Z"
}
```

**Comment by JLockerman** [comment]
This should be fixed by PR #184

**Comment by cevian** [comment]
@wenerme This should be fixed in release 0.1.0-beta.2. Can you please confirm?

**Comment by wenerme** [comment]
Works now.
